core/src/test/resources/schema/postgres/partitioned-schema.sql

```diff
@@ -36,6 +36,7 @@ CREATE TABLE IF NOT EXISTS public.journal
 ) PARTITION BY LIST (persistence_id);
 
 CREATE INDEX journal_ordering_idx ON public.journal USING BRIN (ordering);
+CREATE INDEX journal_tags_idx ON public.journal USING GIN(tags);
```

Comment:
Do you run perf-tests? From my previous tests it look like GIN index does not improve performance in our case.

Reply:
I've been comparing execution plans (query costs)